src/features/event/editor/date-range/popover.tsx

```diff
@@ -32,7 +33,10 @@ export default function DateRangePopover({
       <Popover.Portal>
         <Popover.Content
           sideOffset={10}
-          className="data-[state=open]:data-[side=bottom]:animate-slideUpAndFade data-[state=open]:data-[side=left]:animate-slideRightAndFade data-[state=open]:data-[side=right]:animate-slideLeftAndFade data-[state=open]:data-[side=top]:animate-slideDownAndFade dark:bg-violet z-50 rounded-md border border-gray-300 bg-white p-4 shadow-lg dark:border-gray-400"
+          className={cn(
+            "bg-background z-50 rounded-md border border-gray-300 p-4 shadow-lg dark:border-gray-400",
```

Comment:
```suggestion
            "bg-background z-50 rounded-md border border-gray-400 p-4 shadow-lg",
```
I think it would make sense for this component to have the same style as the selectors.